### ISSUE TITLE
Multiple detection for the same tag

### ISSUE THREAD

**Issue by yusufguleray** [Sologala/pyStag#4]
I have realized that even in a frame that has only a single tag, there are multiple detections of it (3-4) at the same location. Is this an artifact of the wrapper?

**Comment by Sologala** [comment]
Yes, I met this before too. For me , I just use a  NMS to alleviate it.
